src/main/java/org/folio/consortia/service/TenantManager.java

```diff
@@ -68,4 +70,12 @@ public interface TenantManager {
    */
   void deleteIdentityProvider(String memberTenantId);
 
+  /**
+   * Sets up custom keycloak login flow for the central tenant.
+   *
+   * @param consortiumId the consortium id
+   * @param centralTenantId the central tenant id
+   */
+	void setupCustomLogin(UUID consortiumId, String centralTenantId);
```

Comment:
Can add more info about it in java docs
Sets up custom keycloak login flow for the central tenant to be able to login for users created in member tenants.